### PR TITLE
Support initContainers cleaning

### PR DESCRIPTION
**Pull request by maboelnour**
Clean `initContainers` images for:
 - Deployments
 - Statefulsets
 - Cronjobs
 - Jobs
 - Daemonsets
 - Replicasets
 - Replicationcontrollers
 - Pods

### CODE REVIEW

File: kleaner.sh [hunk 1]
```diff
@@ -38,16 +38,25 @@ function get_images() {
...
+    cronjobsinitcont=`kubectl get cronjobs --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.jobTemplate.spec.template.spec.initContainers[*].image}{end}{"\n"}'`
     jobs=`kubectl get jobs --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.containers[*].image}{end}{"\n"}'`
+    jobsinitcont=`kubectl get jobs --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.initContainers[*].image}{end}{"\n"}'`
     daemonsets=`kubectl get daemonsets --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.containers[*].image}{end}{"\n"}'`
+    daemonsetsinitcont=`kubectl get daemonsets --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.initContainers[*].image}{end}{"\n"}'`
     replicasets=`kubectl get replicasets --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.containers[*].image}{end}{"\n"}'`
+    replicasetsinitcont=`kubectl get replicasets --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.initContainers[*].image}{end}{"\n"}'`
```
Comment (hammady) [inline kleaner.sh:51]: Add to the below array.

File: kleaner.sh [hunk 2]
```diff
@@ -38,16 +38,25 @@ function get_images() {
...
     replicationcontrollers=`kubectl get replicationcontrollers --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.containers[*].image}{end}{"\n"}'`
+    replicationcontrollersinitcont=`kubectl get replicationcontrollers --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.template.spec.initContainers[*].image}{end}{"\n"}'`
```
Comment (hammady) [inline kleaner.sh:53]: Add to the below array.

File: kleaner.sh [hunk 3]
```diff
@@ -38,16 +38,25 @@ function get_images() {
...
     pods=`kubectl get pods --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.containers[*].image}{end}{"\n"}'`
+    podsinitcont=`kubectl get pods --all-namespaces -o jsonpath='{range .items[*]}{"\n"}{.spec.initContainers[*].image}{end}{"\n"}'`
```
Comment (hammady) [inline kleaner.sh:55]: Add to the below array.

**Review by hammady — CHANGES_REQUESTED** [review]

File: kleaner.sh (hunk 3 above)
Comment (maboelnour) [reply kleaner.sh:55]: Added,

File: kleaner.sh (hunk 2 above)
Comment (maboelnour) [reply kleaner.sh:53]: Added,

File: kleaner.sh (hunk 1 above)
Comment (maboelnour) [reply kleaner.sh:51]: Added,

**Review by hammady — APPROVED** [review]

**Pull request merged**